pom.xml

```
@@ -16,7 +16,7 @@
   <properties>
     <revision>2</revision>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.361.1</jenkins.version>
+    <jenkins.version>2.440.1</jenkins.version>
```

Can you explain further why the minimum Jenkins version was increased, especially to so recent a version?
```suggestion
    <jenkins.version>2.361.1</jenkins.version>
```